.github/workflows/linux-docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Linux docker-compose
+on:
+  - push
+  - pull_request
+jobs:
+  build:
+    name: Build
```

Comment:
"Test" is better than "Build".

Reply:
I copied your MSVC file ;)

Reply:
I thought so. :-)
The MSVC file just builds and installs. It doesn't run test because we can't run test with Visual Studio generator yet. But this job can run test.